west.yml

```diff
@@ -154,7 +154,7 @@ manifest:
     - name: matter
       repo-path: sdk-connectedhomeip
       path: modules/lib/matter
-      revision: c36fcaf148cf12d28a4e7a976a8aef9727ee73f1
+      revision: 3da44025b18d17adacf0a4abf0456c5735399dbd
```

Comment:
My previous comment was out of date. This change was already pulled by some other PR, that's why CI failed, because you actually step back to the previous commit. To sum up SHA change is not needed at all.